squeezenet-burn/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                              Apache License
+                        Version 2.0, January 2004
+                     http://www.apache.org/licenses/
+
+TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+
```

Comment:
Maybe we can use symlinks instead of copying the licences.

Reply:
Yeah, definitely. I created through sym link. I not sure how come the file got it.